src/murmur/Server.cpp

```diff
@@ -1552,22 +1595,16 @@ void Server::sslError(const QList< QSslError > &errors) {
 	bool ok = true;
 	foreach (QSslError e, errors) {
 		switch (e.error()) {
-			case QSslError::InvalidPurpose:
+			case QSslError::SslError::InvalidPurpose:
 				// Allow email certificates.
 				break;
-			case QSslError::NoPeerCertificate:
-			case QSslError::SelfSignedCertificate:
-			case QSslError::SelfSignedCertificateInChain:
-			case QSslError::UnableToGetLocalIssuerCertificate:
-			case QSslError::UnableToVerifyFirstCertificate:
-			case QSslError::HostNameMismatch:
-			case QSslError::CertificateNotYetValid:
-			case QSslError::CertificateExpired:
-				u->bVerified = false;
-				break;
 			default:
-				log(u, QString("SSL Error: %1").arg(e.errorString()));
-				ok = false;
+				if (qlAllowedSslClientErrors.contains(e.error())) {
+					u->bVerified = false;
+				} else {
+					log(u, QString("SSL Error (%2): %1").arg(e.errorString()).arg((int) e.error()));
```

Comment:
Why the cast to `int`?

Reply:
I wanted that the error message not only contains the error string, but also the number of the error (for quicker lookup when interpreting it).

Reply:
This is not about including the int in the error message - it's about why the explicit cast is necessary because it seems like it should work without it :)